users.go

```diff
@@ -925,9 +925,10 @@ func (s *UsersService) GetUserActivities(opt *GetUserActivitiesOptions, options
 // GitLab API docs:
 // https://docs.gitlab.com/ce/api/users.html#user-status
 type UserStatus struct {
-	Emoji       string `json:"emoji"`
-	Message     string `json:"message"`
-	MessageHTML string `json:"message_html"`
+	Emoji        string `json:"emoji"`
+	Availability string `json:"availability"`
```

Comment:
Is there a fixed list of availability values? If so we should probably make a `const` for it instead of using a `string`

Reply:
good point. I tried to find one (for about 5 minutes :see_no_evil:) and came up empty.
I'll see what gitlab's code has to say about it :+1:

Reply:
ok, it seems gitlab [currently has only two possible options](https://gitlab.com/gitlab-org/gitlab-foss/-/blob/3cfe2ab491cdaf46d5a8ff6359fc6a89a7557eea/app/models/user_status.rb#L22): `not_set` and `busy`.

added as consts :heavy_check_mark: